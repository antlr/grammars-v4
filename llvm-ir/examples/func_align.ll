$g = comdat any
$h = comdat any

define void @f() align 2 {
0:
	ret void
}

define void @g() align 2 comdat {
; <label>:0
	ret void
}

define void @h() comdat align 2 {
; <label>:0
	ret void
}
