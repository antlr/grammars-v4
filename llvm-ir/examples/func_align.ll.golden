$g = comdat any
$h = comdat any

define void @f() align 2 {
0:
	ret void
}

define void @g() comdat align 2 {
0:
	ret void
}

define void @h() comdat align 2 {
0:
	ret void
}
